src/gallery.js

```diff
@@ -20,8 +21,10 @@ export function applyTransform(element, transform) {
   element.style.transform = transform;
 }
 
-export class Gallery {
+export class Gallery extends Eventer {
```

Comment:
https://en.wikipedia.org/wiki/Composition_over_inheritance

Eine Galerie ist für mich keine Spezialisierung eines Event Emitters, von daher hätte ich die `Eventer` Instanz eher in eine Variable geschrieben, als das Verhalten per Inheritance übernommen.